blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java

```diff
@@ -57,8 +57,8 @@
 
```

Comment:
Given how close we are to release, let's leave the classname as-is. We can rename in a future release, handling it with our `deserializingClassRenames.properties` stuff.